packages/storybook8/stories/Composites/Adapters/Docs.mdx

```diff
@@ -0,0 +1,252 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { Note } from '../../utils/Note';
+
+import CallAdapterExampleText from '!!raw-loader!./snippets/CallAdapterExample.snippet.tsx';
+import CallAdapterStateChangesText from '!!raw-loader!./snippets/CallAdapterStateChanges.snippet.txt';
+import ChatAdapterExampleText from '!!raw-loader!./snippets/ChatAdapterExample.snippet.tsx';
+import ChatAdapterStateChangesText from '!!raw-loader!./snippets/ChatAdapterStateChanges.snippet.txt';
+
+<Meta title="Composites/Adapters" />
+
+# Adapters for Composites
+
+Composites deliver turn-key communication experiences for developers.
+They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
+To power Composites, the UI library provides `Adapters`, one for each Composite. These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor to provide developers a single interface to configure Composites.
```

Comment:
nitpick: The original content had
` one for the calling composite [here](./?path=/docs/composites-call-basicexample--basic-example) and one for chat composite
[here](./?path=/docs/composites-chat-basicexample--basic-example).`
Are we ok with the content change here to not include the links?

Reply:
It excluded `callwithchat` so I shortened to just `one for each composite` instead. Can revert if you like